coroutines/README.md

```diff
@@ -1,52 +1,64 @@
 ![badge-js]
 ![badge-jvm]
+![badge-mac]
 
 # Coroutines
 
 Toolbox of utilities for [Coroutines].
 
-## Installation
+## Setup
 
 ### Gradle
 
-Artifacts are hosted on GitHub packages, which can be configured as follows:
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.juul.tuulbox/coroutines/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.juul.tuulbox/coroutines)
+
+Coroutines toolbox can be configured via Gradle Kotlin DSL as follows:
+
+#### Multiplatform
 
 ```kotlin
-import java.net.URI
+plugins {
+    id("com.android.application") // or id("com.android.library")
+    kotlin("multiplatform")
+}
 
 repositories {
-    maven {
-        url = URI("https://maven.pkg.github.com/juullabs/android-github-packages")
-        credentials {
-            username = findProperty("github.packages.username") as String
-            password = findProperty("github.packages.password") as String
-        }
-    }
+    jcenter() // or mavenCentral()
 }
-```
-
-Then the needed artifact(s) can be defined as dependencies.
-
-**Multiplatform projects**
 
-```kotlin
 kotlin {
+    android()
+    js().browser() // and/or js().node()
+    macosX64()
+
     sourceSets {
         val commonMain by getting {
-            implementation("com.juul.tuulbox:coroutines:$version")
+            dependencies {
+                implementation("com.juul.tuulbox:coroutines:$version")
+            }
         }
     }
 }
+
+android {
+    // ...
+}
 ```
 
-**Platform-specific projects**
+#### Platform-specific
 
 ```kotlin
+repositories {
+    jcenter() // or mavenCentral()
+}
+
 dependencies {
     implementation("com.juul.tuulbox:coroutines-$platform:$version")
 }
 ```
 
+_Where `$platform` represents (should be replaced with) the desired platform dependency (e.g. `android`)._
```

Comment:
Should this actually say `jvm`? I don't think we have any android-specific compilation yet.

Reply:
Good call. I was blindly copy/pasting from other projects. I'll change it to `jvm` here. 👍